Libraries/Components/SliderIOS/SliderIOS.ios.js

```diff
@@ -76,6 +76,13 @@ var SliderIOS = React.createClass({
      */
     disabled: PropTypes.bool,
 
+
+    /**
+     * Sets an image for the thumb. It only supports images that are included as assets. It expects a dictionary that has the following form: {'source':'mySource','width':0,'height':0}
+     */
+    thumbImage: PropTypes.object,
```

Comment:
@nicklockwood I am working on making the suggested changes. As per your suggestion, to include a thumbImage, it would be like this <SliderIOS thumbImage={require('image!SliderBar')}>